### PR TITLE
make more of the shell pass lints

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Ref #72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xichengliudui** [comment]
/assign @BenTheElder @fejta 
update https://github.com/kubernetes/kubernetes/pull/74228

File: cluster/kube-up.sh [hunk 1]
```diff
@@ -84,6 +84,7 @@ elif [[ "${validate_result}" == "2" ]]; then
 fi
 
 if [[ "${ENABLE_PROXY:-}" == "true" ]]; then
+  # shellcheck disable=SC1091
```
Comment (xichengliudui) [inline cluster/kube-up.sh]: why do I disable it ref: https://github.com/koalaman/shellcheck/wiki/SC1091

Comment (spiffxp) [reply cluster/kube-up.sh]: Thanks for the link!

I question whether this entire block is necessary
- https://cs.k8s.io/?q=ENABLE_PROXY&i=nope&files=&repos=
- https://cs.k8s.io/?q=/tmp/kube-proxy-env&i=nope&files=&repos=

As far as I can tell, it used to be used by cluster/openstack-heat/util.sh

**Review by spiffxp — APPROVED** [review]
/lgtm
/approve

**Pull request merged**